scripts/i18n-utils/i18n-utils.js

```diff
@@ -9,7 +9,8 @@ const srcContents = fs.readFileSync(srcFile).toString();
 const baseLangContents = {};
 
 const strings = [];
-const regex = /\b(?:i18n`(.*?)`)/g;
+// @see https://regexr.com/6jtib
+const regex = /\b(?:i18n\(\s*['"`](.*?)['"`]\s*[,)])/g;
```

Comment:
👍. Would be nice if we could avoid having to update this any time an api change happens, but we can let that be future us's problem.